app/views/textile/form.js.erb

```diff
@@ -0,0 +1,22 @@
+<% form = render partial: 'textile/form' %>
+
+$('.textile-form').removeClass('loading-indicator').html('');
+$('.textile-form').removeClass('loading-indicator').append('<%= escape_javascript(form) %>');
+
+$('[data-behavior~=delete-field]').click( function(){
+  $(this).closest('[data-behavior~=textile-form-field]').remove();
+});
+
+$('[data-expand~=auto]').each( function() {
+  $(this).css({'padding': '0.375rem 0.75rem', 'height': this.scrollHeight});
+});
+
+$('[data-expand~=auto]').on('keyup', function(e) {
```

Comment:
There's a lot happening here. I can't follow the purpose.

Textareas have their height set to scrollHeight. and then to 1px, and then back to scrollheight?
What's the reasoning?

Reply:
Gonna have to phone a friend for this one. @MattBudz ?

Reply:
This is for cross-browser compatibility for automatically expanding and shrinking textareas as the user types. If the height isn't set it to 1px first then the browser can't set it to the scroll height properly and it also prevents the textbox from shrinking when the user is backspacing.